packages/mds-daily/tests/request-handlers.test.ts

```diff
@@ -16,9 +16,9 @@ describe('Request handlers', () => {
         } as any
       }
       await getRawTripData(
-        {
+        ({
           params: { trip_id: 'fake-trip-id' }
-        } as DailyApiRequest,
+        } as unknown) as DailyApiRequest,
```

Comment:
Uhhg, not a fan of this `unknown` skullduggery. Hopefully it'll go away as I migrate to mocking full request and response objects with Sinon.